felix/bpf/tc/attach.go

```diff
@@ -628,21 +634,51 @@ func (ap *AttachPoint) setMapSize(m *libbpf.Map) error {
 }
 
 // nolint
-func updateJumpMap(obj *libbpf.Obj, isHost bool) error {
-	if !isHost {
-		err := obj.UpdateJumpMap("cali_jump", string(policyProgram), PolicyProgramIndex)
+func updateJumpMap(obj *libbpf.Obj, isHost bool, ipv6Enabled bool) error {
+	ipVersions := set.New()
+	ipVersions.Add("IPv4")
+	if ipv6Enabled {
+		ipVersions.Add("IPv6")
+	}
+
+	ipVersions.Iter(func(ipFamily interface{}) error {
```

Comment:
If something fails with `Iter()` method, like for example if one of the updates fails, the program panics. I am not sure if it is OK here. For instance, I noticed an error here stops the the `ut` target with a panic error.